internal/tss/channel.go

```diff
@@ -0,0 +1,34 @@
+package tss
+
+type RPCGetIngestTxResponse struct {
+	// a status that indicated whether this transaction failed or successly made it to the ledger
+	Status string
+	// the raw TransactionEnvelope XDR for this transaction
+	EnvelopeXdr string
+	// the raw TransactionResult XDR of the envelopeXdr
+	ResultXdr string
+	// The unix timestamp of when the transaction was included in the ledger
+	CreatedAt int64
+}
+
+type RPCSendTxResponse struct {
+	// the status of an RPC sendTransaction call. Can be one of [PENDING, DUPLICATE, TRY_AGAIN_LATER, ERROR]
+	Status string
```

Comment:
```suggestion
	Status RPCTXStatus
```

When working with string enums it's best to define a type for it:
```
type RPCTXStatus string

const (
	PendingStatus       RPCTXStatus = "PENDING"
	DuplicateStatus     RPCTXStatus = "DUPLICATE"
	TryAgainLaterStatus RPCTXStatus = "TRY_AGAIN_LATER"
	ErrorStatus         RPCTXStatus = "ERROR"
)
```
Then you can also define a validation method for its values
```
func (s RPCTXStatus) IsValid() bool {
	switch s {
	case
		PendingStatus,
		DuplicateStatus,
		TryAgainLaterStatus,
		ErrorStatus:
		return true
	default:
		return false
	}
}
```

Reply:
I like it, added statuses for both sendTransaction and getTransaction RPC calls